framework/src/main/resources/config.conf

```diff
@@ -672,27 +683,72 @@ vm = {
   minTimeRatio = 0.0
   maxTimeRatio = 5.0
   saveInternalTx = false
+  # lruCacheSize = 500
+  # vmTrace = false
 
-  # Indicates whether the node stores featured internal transactions, such as freeze, vote and so on
+  # Indicates whether the node stores featured internal transactions, such as freeze, vote and so on. default false.
   # saveFeaturedInternalTx = false
 
-  # Indicates whether the node stores the details of the internal transactions generated by the
-  # CANCELALLUNFREEZEV2 opcode, such as bandwidth/energy/tronpower cancel amount.
+  # Indicates whether the node stores the details of the internal transactions generated by the CANCELALLUNFREEZEV2 opcode,
+  # such as bandwidth/energy/tronpower cancel amount. default false.
   # saveCancelAllUnfreezeV2Details = false
 
   # In rare cases, transactions that will be within the specified maximum execution time (default 10(ms)) are re-executed and packaged
   # longRunningTime = 10
 
-  # Indicates whether the node support estimate energy API.
+  # Indicates whether the node support estimate energy API. default false.
   # estimateEnergy = false
 
-  # Indicates the max retry time for executing transaction in estimating energy.
+  # Indicates the max retry time for executing transaction in estimating energy. default 3.
   # estimateEnergyMaxRetry = 3
 }
 
 committee = {
   allowCreationOfContracts = 0  //mainnet:0 (reset by committee),test:1
   allowAdaptiveEnergy = 0  //mainnet:0 (reset by committee),test:1
+  # allowCreationOfContracts = 0
```

Comment:
It's advisable to include a warning: These parameters are designed for private chain testing only and cannot be freely switched on or off in production systems.